zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedKubernetesClusterChecker.scala

```diff
@@ -33,7 +33,7 @@ object DeletedKubernetesClusterChecker {
           _ <- if (isDryRun) F.unit
           else
             clusterOpt match {
-              case None    => dbReader.updateK8sClusterStatus(cluster.id)
+              case None    => dbReader.markK8sClusterDeleted(cluster.id) >> dbReader.unlinkPDFromK8sCluster(cluster.id)
```

Comment:
maybe we can just null `diskid` in `markK8sClusterDeleted` query?

Reply:
Hmm I can revisit and try that if you think that'd be better, my thought was to keep the queries separate to be the most "single purpose" but I am not very strongly opinionated on it.

Reply:
I agree in general, but I think in this case, whenever we mark a cluster deleted, we probably do want to unlink disk as well

Reply:
Okay I have combined the queries to unlink and delete in the same query for each K8s cluster and runtimes